Digipost.Signature.Api.Client.Core/Identifier.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Digipost.Signature.Api.Client.Core
+{
+    public class Identifier
+    {
+        public Identifier(string value)
+        {
+            Value = value;
+        }
+
+        public string Value { get; }
+
+        public override bool Equals(object obj)
+        {
+            var other = obj as Identifier;
```

Comment:
(det gjelder forøvrig generelt for `Equals`-metoder. Usikker på hva som er vanlig i .NET-land her :)